src/Compilers/CSharp/Test/Emit/PDB/PDBTests.cs

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
Is the encoding change necessary?

---
In reply to: [98543858](https://github.com/dotnet/roslyn/pull/16803#discussion_r98543858) [](ancestors = 98543858)